.travis.yml

```diff
@@ -2,18 +2,31 @@ language: php
 
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
Does stale packagist metadata also invalidate the files then? I've updated the code, but if possible, I'm very interested in a source for this (as I use this exact code in multiple repositories).